package.json

```diff
@@ -10,7 +10,8 @@
   "main": "./server/index.js",
   "scripts": {
     "create-superuser": "NODE_ENV=production lvconnect create-superuser",
-    "dev:back": "APP_ENV=dev nodemon . --watch ./packages/api",
+    "dev": "run-p dev:front dev:back",
+    "dev:back": "APP_ENV=dev nodemon . --watch ./packages/api --watch ./packages/config",
```

Comment:
watch for changes in `packages/config` (the adding of the `LEAD` role didn't reload nodemon)